metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -108,12 +108,19 @@ final class Doctor(
       isSemanticdbEnabled: Boolean,
       scala: ScalaTarget
   ): String = {
+
+    def isMaven: Boolean = workspace.resolve("pom.xml").isFile
+    def hint() =
+      if (isMaven)
+        "check how to enable semanticDB on the " +
+          "<a href=https://scalameta.org/metals/docs/build-tools/overview.html>metals website</a>"
```

Comment:
```suggestion
          "<a href=https://scalameta.org/docs/semanticdb/guide.html#producing-semanticdb>Scalameta website</a>"
```

Reply:
I was thinking of making Maven specific instructions for this. There is nothing on the scalameta website about maven.

Reply:
Should we create a Maven-specific page in that case? Similar to the sbt page https://scalameta.org/metals/docs/build-tools/sbt.html

Reply:
That's my thinking yes.